package.json

```diff
@@ -162,19 +170,19 @@
   "author": "Mark Jaquith <mark@jaquith.me>",
   "license": "MIT",
   "peerDependencies": {
-    "svelte": "^4.0.0||^5.0.0",
-    "@sveltejs/kit": "^1.25.1||^2.0.2"
+    "svelte": "^4.0.0 || ^5.0.0"
```

Comment:
Now that Svete v5 is stable, it should be safe to remove v3 as peer dependency 🫡